typing/env.ml

```diff
@@ -703,20 +703,29 @@ let find_name_module ~mark name tbl =
 
 let add_persistent_structure id env =
   if not (Ident.persistent id) then invalid_arg "Env.add_persistent_structure";
-  if not (Current_unit_name.is_name_of id) then
-    let summary =
+  if Current_unit_name.is_name_of id then env
+  else begin
+    let material =
+      (* Addition only observably changes the enivonment if it shadows a
+         non-persistent module already in the environment. *)
```

Comment:
there is a typo in this comment (`enivonment`), and I don't really understand the sentence, and it would be nice to git the issue numbers of the two related PRs.

Reply:
I'll fix the typo and reference the PRs, but I'll need a more precise description of what you don't understand about the sentence.

Reply:
(the edited version has grammat that I can understand, so this is resolved.)